### ISSUE TITLE
CreateJS support ? 

### ISSUE THREAD

**Issue by tranlong021988** [DragonBones/DragonBonesJS#45]
Hi, I'm new with DragonBone but notice that we only have Egret and Pixi support, so is there any plan for CreateJS ?

**Comment by akdcl** [comment]
We do not have such a plan for the time being, the manpower is limited, but it seems that CreateJS does not have Mesh? So it seems unable to support the skinning animation.

**Comment by tranlong021988** [comment]
Yes, CreateJS have no mesh, and Pixi is super star right now, but there is new feature called StageGL which help improve performance by intergrading WebGL to CreateJs, it's still in beta but look promising, they have WebGL and Adobe Animate as scene editor. 
Anw, I'm deploying some tests with this: 
https://github.com/06wj/DragonBonesJS
But it's seem no update anymore although having CreateJS support. Hopefully we will have CreateJS officially support for CreateJS in future. That may be awesome.

**Comment by akdcl** [comment]
Thank you for your advice, we'll try to support more engines as much as possible. :)
